planning_group: "manipulator" # manipulator planning group
manipulator_base_link: "base_link" # base link of the manipulator
manipulator_reach: 1.30 # maximum reach of the manipulator - this will set the radius of the initial workspace sphere where all positions will be calculated in
resolution: 0.05 # resolution of the calculated workspace in meters
scale: 0 # 0 means automatic scaling, otherwise the given value will be used
publish_rate: 1 # rate in Hz in which the workspace points will be published
#file_name: "" # you could provide a file name with suffix .bag (like test.bag) for the generated workspace
continue_calculation: false # the node will look if there is a file with the same filename and .partial and will continue the calculation, make sure to provide the same config values here before starting the continuation!

# Optional: You can provide a fixed end-effector orientation (as quaternion), to ignore the orientation, comment this parameters, moveit will use identity as orientation
ee_orientation_x: 0
ee_orientation_y: 1
ee_orientation_z: 0
ee_orientation_w: 0

orientation_tolerance: 0.01 #orientation tolerance, to calculate the reachable workspace set this to a high value, default is 0.01
# Optional: You can provide a detailed region to inspect that in more detail, if one of the values is not provided min and max are set to "0" and "manipulator_reach". If no value is set it will use a spherical volume of "manipulator_reach" size
#x_min: -0.3
#x_max: 0.2
#y_min: -0.1
#y_max: 0.1
z_min: -0.1
z_max: -0.1
complete_region: false # Optional (use only with region parameters) Complete region defined by x,z,z ranges (true) or just the region inside the manipulator_reach sphere (false)

# Define a resolution for the initial poses which will be tested
resolution_initial_poses: 0.05
